README.md

```diff
@@ -1,2 +1,51 @@
 # UCWifiConnect
 Snap to be able to switch a wireless card of a UC device into AP mode and use it to configure wireless
+
+## Development Environment
+
+### Install snapd and snapcraft
+```bash
+sudo apt install snapd snapcraft
+
+# Verify snapcraft is installed ok by printing out the version
+snapcraft -v
+2.27.1
+```
+should output current version. More information on [snapcraft.io](https://snapcraft.io)
+
+### Install Go
+Follow the instructions to [install Go](https://golang.org/doc/install).
+
+- Install NVM
```

Comment:
mmm.. browsers by default don't render markdown. Github does. You can see it at https://github.com/rmescandon/UCWifiConnect/blob/look-and-feel/README.md.
Also, If you use Visual Studio Code as IDE, you can switch between edit and view views by pressing Ctrl+Alt+V.
In any case, it is true that i missed one header for "web development" section, just added